### ISSUE TITLE
[BUG] 0 Bytes file when exporting as "Same as input" resolution

### ISSUE THREAD

**Issue by limosoft-de** [limosoft-de/raffytimelapse#8]
**Describe the bug**
When exporting the timelapse as "Same as input" resolution, Raffy Timelapse creates a 0 bytes file instead of the correct timelapse.

**To Reproduce**
1. Click on 'Start'
2. Select resolution 'Same as input'
3. Select the high quality profile
4. The other settings don't seem to have an effect on this bug and can be set any way
5. Start rendering by clicking 'OK'
6. Raffy exports 0 bytes file

**Expected behavior**
Regular timelapse being exported

**Screenshots**
none

**Symstem Information**
 - OS: Windows 10
 - Raffy Timelapse Version: 2.1.0

**Additional context**
Reported by Arménio via email

**Comment by limosoft-de** [comment]
**Update 2019-06-19 16:00 CEST**

This issue isn't persistent and doesn't happen everytime. Reason couldn't be found yet. Under further investigation...

**Comment by limosoft-de** [comment]
**Update 2019-06-20 16:45 CEST**

This bug seems to be an issue caused by extreme high resolution images, either extending the system capabilities or the H.264 codec capabilities.

Converter output:
```
x264 [error]: malloc of size 80190656 failed
Video encoding failed
Conversion failed!
```

For now, using one of the default resolutions or lower quality input images will help avoid this bug. Will be fixed in a feature release.

**Comment by limosoft-de** [comment]
Added FFmpeg 64bit support. Will be fixed with the next update.

**Comment by limosoft-de** [comment]
Fixed in V2.2.0
